<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1291</width>
      <height>631</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"> QTabBar::tab {
  background: gray;
  color: white;
  padding: 10px;
 }

 QTabBar::tab:selected {
  background: lightgray;
 }

QWidget#tab_5
{
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 3, 53, 255), stop:1 rgba(108, 108, 108, 255));
}

QWidget#tab_6
{
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 3, 53, 255), stop:1 rgba(108, 108, 108, 255));
} 

QWidget#tab_7
{
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 3, 53, 255), stop:1 rgba(108, 108, 108, 255));
}

QWidget#options
{
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 3, 53, 255), stop:1 rgba(108, 108, 108, 255));
}</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Test Overlay&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Overlay</string>
     </attribute>
     <attribute name="toolTip">
      <string>Overlay two images of the same size</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Select an image to apply an ovlerlay to</string>
      </property>
      <property name="text">
       <string>Original Image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>20</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Select an image to overlay</string>
      </property>
      <property name="text">
       <string>Overlay Image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Overlay the two above images</string>
      </property>
      <property name="text">
       <string>Overlay</string>
      </property>
     </widget>
     <widget class="QLabel" name="overlayed">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>320</y>
        <width>291</width>
        <height>241</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Overlayed Image</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="original_image">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>281</width>
        <height>241</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Original Image</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="overlay_image">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>50</y>
        <width>281</width>
        <height>241</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Overlay Image</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Test EQ&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Equalize</string>
     </attribute>
     <attribute name="toolTip">
      <string>Appl histogram equalization to an image</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>141</width>
        <height>71</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Select an image to equalize</string>
      </property>
      <property name="text">
       <string>Original Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="orig_eq_image">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>501</width>
        <height>361</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Original Image</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>60</y>
        <width>141</width>
        <height>71</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Equialize the image on the left</string>
      </property>
      <property name="text">
       <string>Equalize Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="eq_image">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>180</y>
        <width>501</width>
        <height>361</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Equalized Image</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Editor</string>
     </attribute>
     <attribute name="toolTip">
      <string>Edit an image's brightness and contrast</string>
     </attribute>
     <widget class="QLabel" name="adjustImage">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>761</width>
        <height>411</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Edited Image</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>470</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Load an image to edit</string>
      </property>
      <property name="text">
       <string>Load Image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>530</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Reset the image to its original state</string>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>530</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Save the above image</string>
      </property>
      <property name="text">
       <string>Save Image</string>
      </property>
     </widget>
     <widget class="QSlider" name="brightnessSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>451</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Brightness Adjuster</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="sliderPosition">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
     </widget>
     <widget class="QSlider" name="contrastSlider">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>480</y>
        <width>451</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Contrast Adjuster</string>
      </property>
      <property name="minimum">
       <number>900</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="sliderPosition">
       <number>950</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
     </widget>
     <widget class="QLabel" name="brightnessLabel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>510</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Brightness</string>
      </property>
     </widget>
     <widget class="QLabel" name="contrastLabel">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>500</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> color: white;</string>
      </property>
      <property name="text">
       <string>Contrast</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="options">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"> QTabBar::tab {
  background: gray;
  color: white;
  padding: 10px;
 }

 QTabBar::tab:selected {
  background: lightgray;
 }

QWidget#tab_5
{
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 3, 53, 255), stop:1 rgba(108, 108, 108, 255));
}

QWidget#tab_6
{
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 3, 53, 255), stop:1 rgba(108, 108, 108, 255));
} 

QWidget#tab_7
{
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 3, 53, 255), stop:1 rgba(108, 108, 108, 255));
}</string>
     </property>
     <attribute name="title">
      <string>Options</string>
     </attribute>
     <attribute name="toolTip">
      <string>Set options</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Enable Serial Communication</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>COM4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>COM Port</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Baud Rate</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>115200</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1286</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuECE_484W_Milestone_2">
    <property name="title">
     <string>ECE 484W Milestone 2</string>
    </property>
   </widget>
   <addaction name="menuECE_484W_Milestone_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
